hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRequest.java

```diff
@@ -134,7 +135,9 @@ public void setup() throws Exception {
     when(ozoneManager.getMetadataManager()).thenReturn(omMetadataManager);
     when(ozoneManager.getConfiguration()).thenReturn(ozoneConfiguration);
     OMLayoutVersionManager lvm = mock(OMLayoutVersionManager.class);
-    when(lvm.getMetadataLayoutVersion()).thenReturn(0);
+    when(lvm.getMetadataLayoutVersion()).thenReturn(5);
+    when(lvm.getFeature(any())).thenReturn(OMLayoutFeature.SNAPSHOT_SUPPORT);
+    when(lvm.isAllowed(anyString())).thenReturn(true);
```

Comment:
hm, yeah in TestOMSnapshotCreateRequest, TestOMSnapshotDeleteRequest, TestOMSnapshotPurgeRequestAndResponse I need to keep this line 'when(lvm.isAllowed(anyString())).thenReturn(true)', otherwise the aspectJ side of code 
https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/OMLayoutFeatureAspect.java#L84 throws a null pointer exception. Other lines of mock I've removed & updated pr! Thanks ethan.